src/sql/migrations.rs

```diff
@@ -549,6 +549,23 @@ DO UPDATE SET rfc724_mid=excluded.rfc724_mid,
         )
         .await?;
     }
+    if dbversion < 84 {
+        info!(context, "[migration] v84");
+        sql.execute_migration(
+            r#"CREATE TABLE smtp (
+id INTEGER PRIMARY KEY,
+rfc724_mid TEXT NOT NULL,          -- Message-ID
+mime TEXT NOT NULL,                -- SMTP payload
```

Comment:
I'm *kind* of regretting making this comment as now we'll always have to generate 7-bit email.  Given this is the send path it probably is fine to make this commitment, as it makes us conservative in what we send which is probably the right thing to do as we have no control over the 8-bit cleanliness of the SMTP server.  So let's keep it this way, but I felt it worth pointing out we're accepting being 7-bit on the SMTP path with this change.

Reply:
If we want to support binary MIME later, we can delete `smtp` table and recreate it from scratch. Unlike adding columns to `msgs` table, adding tables is cheap, especially if they are small.

Reply:
This is unlikely to ever happen though. The only path to get rid of base64 overhead I see is to add compression on top, with IMAP COMPRESS extension for reception and BURL (RFC 4468) for the sender side.